### DISCUSSION TITLE
How do I give pull request permission for a branch and revoke it for the other one

### DISCUSSION THREAD

**Discussion by stormbeforesunsetbee** [community/community#23998] [New to GitHub]
<p>Hi! I wanna set my <code>main</code>/<code>master</code> branch to not allow pull requests to my contributors, so they have to do a pull request on a specific branch to an alpha feature/version development instead. I coudn’t find the setting here in <a href="https://github.com/my-username/my-repo/settings/branches" rel="noopener nofollow ugc">https://github.com/my-username/my-repo/settings/branches</a> anyways</p>

**Comment by airtower-luna** [comment, marked as answer]
<p>I don’t think there is an option for that. But you could write an Actions workflow that either closes the request with a comment, or (nicer) retargets it to a more appropriate branch.</p>
<aside class="onebox githubfolder">
  <header class="source">
      <img src="https://docs.github.com/assets/images/site/favicon.svg" class="site-icon" width="16" height="16">
      <a href="https://docs.github.com/en/rest/reference/pulls#update-a-pull-request" target="_blank" rel="noopener">docs.github.com</a>
  </header>
  <article class="onebox-body">
    

<h3><a href="https://docs.github.com/en/rest/reference/pulls#update-a-pull-request" target="_blank" rel="noopener">Pulls - GitHub Docs</a></h3>

<p><a href="https://docs.github.com/en/rest/reference/pulls#update-a-pull-request" target="_blank" rel="noopener">//docs.github.com/en/rest/reference/pulls</a></p>


  </article>
  <div class="onebox-metadata">
    
    
  </div>
  <div style="clear: both"></div>
</aside>

**Comment by stormbeforesunsetbee** [comment]
<p>Thanks for answering!</p>
<p>Anyways, how do I edit my post? I wanna change that sentence to *specific branch <strong>for</strong> alpha …</p>

**Comment by airtower-luna** [comment]
<p>There should be a pencil symbol near the bottom right of your post (second symbol left of “Reply”):</p>
<p><img src="https://user-images.githubusercontent.com/5814919/181080565-48cdfd5f-6ccd-430e-9b2a-60f2390139c3.png" alt="grafik" data-base62-sha1="paaLRSTahGBZ1j63mjkA7dYdxeh" width="321" height="61"></p>
